pom.xml

```diff
@@ -16,6 +16,123 @@
+  </properties>
+
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.mule.tools.maven</groupId>
+        <artifactId>mule-maven-plugin</artifactId>
+        <version>${mule.maven.plugin.version}</version>
+        <extensions>true</extensions>
+      </plugin>
+
+      <plugin>
+        <artifactId>maven-resources-plugin</artifactId>
+        <version>3.1.0</version>
+        <executions>
+          <execution>
+            <id>copy-munit-resources</id>
+            <phase>process-test-resources</phase>
+            <goals>
+              <goal>copy-resources</goal>
+            </goals>
+            <configuration>
+              <outputDirectory>${basedir}/target/test-mule/munit</outputDirectory>
+              <resources>
+                <resource>
+                  <directory>src/test/munit</directory>
+                  <filtering>true</filtering>
+                </resource>
+              </resources>
+            </configuration>
+          </execution>
+        </executions>
+      </plugin>
+
+      <plugin>
+        <groupId>com.mulesoft.munit.tools</groupId>
+        <artifactId>munit-maven-plugin</artifactId>
+        <version>${munit.version}</version>
+        <executions>
+          <execution>
+            <id>test</id>
+            <phase>test</phase>
+            <goals>
+              <goal>coverage-report</goal>
+            </goals>
+          </execution>
+        </executions>
+        <configuration>
+          <runtimeProduct>MULE_EE</runtimeProduct>
+          <runtimeVersion>${app.runtime}</runtimeVersion>
+          <redirectTestOutputToFile>true</redirectTestOutputToFile>
+          <coverage>
+            <runCoverage>true</runCoverage>
+            <failBuild>false</failBuild>
+            <requiredApplicationCoverage>50</requiredApplicationCoverage>
+            <requiredResourceCoverage>50</requiredResourceCoverage>
+            <requiredFlowCoverage>50</requiredFlowCoverage>
+            <formats>
+              <format>console</format>
+              <format>html</format>
+              <format>json</format>
+              <format>sonar</format>
+            </formats>
+          </coverage>
+        </configuration>
+      </plugin>
+
+      <plugin>
+        <groupId>com.mulesoft.munit</groupId>
+        <artifactId>munit-extensions-maven-plugin</artifactId>
+        <version>${munit.extensions.maven.plugin.version}</version>
+        <executions>
+          <execution>
+            <goals>
+              <goal>test</goal>
+            </goals>
+            <phase>test</phase>
+          </execution>
+        </executions>
+        <configuration>
+          <argLines>
+            <!--suppress UnresolvedMavenProperty -->
+            <argLine>
+              -javaagent:${settings.localRepository}/org/jacoco/org.jacoco.agent/${jacoco.maven.plugin.version}/org.jacoco.agent-${jacoco.maven.plugin.version}-runtime.jar=destfile=${session.executionRootDirectory}/target/jacoco.exec
```

Comment:
Hoping a lot of this can go away too as jacoco is only for coverage.

Reply:
Removed the "argLines" element. The rest of this plugin is required.